### PR TITLE
Update property-info to v6

### PR DESCRIPTION
**Pull request by ErikZigo**
Part 1 https://keboola.atlassian.net/browse/PST-1257

testy v naší k8s facade https://github.com/keboola/platform-libraries/runs/22520079101 a sandbox komponentě https://github.com/keboola/sandboxes/runs/22521632605 doběhly ok

### CODE REVIEW

File: src/AbstractModel.php [hunk 1]
```diff
@@ -165,7 +165,9 @@ protected function parseProperty($index, $value)
             foreach ($propertyTypes as $PropertyType) {
                 if ($PropertyType->isCollection()) {
                     $values = [];
-                    if (true == ($className = $PropertyType->getCollectionValueType()->getClassName())) {
```
Comment (ErikZigo) [inline src/AbstractModel.php:168]: `getCollectionValueType` je depracated

File: src/Guzzle/Middleware.php [hunk 2]
```diff
@@ -33,7 +34,7 @@ public static function setPatchOperation(): callable
 
                         unset($requestBody['patchOperation']);
 
-                        $modify['body'] = Utils::streamFor(\GuzzleHttp\json_encode($requestBody));
```
Comment (ErikZigo) [inline src/Guzzle/Middleware.php:36]: bylo deprecated

File: composer.json [hunk 3]
```diff
@@ -14,16 +14,19 @@
...
     "require": {
-        "php": "^7.2.5||^8.0",
+        "php": "^8.1",
         "ext-json": "*",
         "guzzlehttp/guzzle": "^6|^7",
         "monolog/monolog": "@stable",
-        "symfony/property-info": "^4|^5",
-        "phpdocumentor/reflection-docblock": "^3.0|^4.0|^5.0"
+        "phpdocumentor/reflection-docblock": "^3.0|^4.0|^5.0",
+        "symfony/property-info": "^5|^6"
```
Comment (ErikZigo) [inline composer.json:22]: povoleni verze 6

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+version: "3.4"
+
+services:
```
Comment (ErikZigo) [inline docker-compose.yml:3]: abych vubec mohl lokalne udelat `composer install`

**Comment by ErikZigo** [comment]
Na nejake zakladni CI jsem zalozil samostatne issue https://keboola.atlassian.net/browse/PST-1526

**Comment by ErikZigo** [comment]
@AdamVyborny 🙏

**Review by AdamVyborny — APPROVED** [review]
👍

**Pull request merged**